cpp/src/io/parquet/bloom_filter_reader.cu

```diff
@@ -163,108 +162,6 @@ struct bloom_filter_caster {
   }
 };
 
-/**
```

Comment:
Now declared in `reader_impl_helpers.hpp` and defined at the bottom of this file.